.github/actions/bc-lint/action.yml

```diff
@@ -0,0 +1,65 @@
+name: 'BC Lint Action'
+description: 'A reusable action for running the BC Lint workflow. See tools/stronghold/README.md for more information.'
+inputs:
+  repo_path:
```

Comment:
Why don't we just take the repo name as argument, and run the checkout of the specified repo in this reusable workflow (this is what we do in the Nova reusable workflows)? Makes the caller workflow cleaner as well since they don't have to checkout and pass around paths themselves.